src/ng/plugins/visibility/visibility.js

```diff
@@ -14,8 +14,16 @@ TemplatePath
 class Visibility extends PluginComponent('qgrid.plugins.visibility.tpl.html') {
 	constructor() {
 		super(...arguments);
+		this.toggle = new Command({
+			execute: key => true,
+			canExecute: () => true
+		});
 
+	}
 
+	get items() {
+		let data = this.model.visibility();
```

Comment:
if variable is not reassignet use const
rename data to visbilityState, it became common name pattern